src/components/button/button.svelte

```diff
@@ -256,7 +284,7 @@
     --color-loading: var(--color);
     --box-shadow-hover: none;
 
-    &:disabled {
+    &:disabled:not(.isLoading) {
```

Comment:
@fallaciousreasoning or @petemill, does this need special treatment for `:host:disabled`? I'm not well versed in the constraints of styling web components.

Reply:
Based on playing with the React and plain HTML examples, this should be fine.

In fact, I'm not seeing the `:host:disabled ...` selector being the active selector in the rule either. Might not be fully understanding on when that would be used.

Reply:
I think because `LeoButton` extends `HTMLElement` not `HTMLButtonElement` it won't have a disabled state anyway.